<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>825</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>825</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #050505;
font: 20px &quot;Funnel Sans&quot;;</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="sideMenu">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>20</number>
       </property>
       <property name="topMargin">
        <number>40</number>
       </property>
       <property name="bottomMargin">
        <number>20</number>
       </property>
       <item>
        <widget class="QPushButton" name="mainPageBtn">
         <property name="minimumSize">
          <size>
           <width>160</width>
           <height>40</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
background-color: #C41E3D;
border-radius: 7px;
color: white;
font: 20px &quot;Funnel Sans Semibold&quot;;
}

QPushButton::hover{
background-color: #FF2C55;
}

QPushButton::pressed{
background-color: #C41E3D;
}</string>
         </property>
         <property name="text">
          <string>Main Page</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Minimum</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>10</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="newSetBtn">
         <property name="minimumSize">
          <size>
           <width>160</width>
           <height>40</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
background-color: #1B1B1B;
border-radius: 7px;
color: white;
font: 20px &quot;Funnel Sans Light&quot;;
}

QPushButton::hover{
background-color: #2D2D2D;
}

QPushButton::pressed{
background-color: #1B1B1B;
}</string>
         </property>
         <property name="text">
          <string>New set</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="newGroupBtn">
         <property name="minimumSize">
          <size>
           <width>160</width>
           <height>40</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
background-color: #1B1B1B;
border-radius: 7px;
color: white;
font: 20px &quot;Funnel Sans Light&quot;;
}

QPushButton::hover{
background-color: #2D2D2D;
}

QPushButton::pressed{
background-color: #1B1B1B;
}</string>
         </property>
         <property name="text">
          <string>New group</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_4">
         <property name="minimumSize">
          <size>
           <width>160</width>
           <height>40</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
background-color: #1B1B1B;
border-radius: 7px;
color: white;
font: 20px &quot;Funnel Sans Light&quot;;
}

QPushButton::hover{
background-color: #2D2D2D;
}

QPushButton::pressed{
background-color: #1B1B1B;
}</string>
         </property>
         <property name="text">
          <string>Placeholder</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_5">
         <property name="minimumSize">
          <size>
           <width>160</width>
           <height>40</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
background-color: #1B1B1B;
border-radius: 7px;
color: white;
font: 20px &quot;Funnel Sans Light&quot;;
}

QPushButton::hover{
background-color: #2D2D2D;
}

QPushButton::pressed{
background-color: #1B1B1B;
}</string>
         </property>
         <property name="text">
          <string>Placeholder</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>160</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="settingsBtn">
         <property name="minimumSize">
          <size>
           <width>160</width>
           <height>40</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
background-color: #1B1B1B;
border-radius: 7px;
color: white;
font: 20px &quot;Funnel Sans Light&quot;;
}

QPushButton::hover{
background-color: #2D2D2D;
}

QPushButton::pressed{
background-color: #1B1B1B;
}</string>
         </property>
         <property name="text">
          <string>Settings</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="mainZone" native="true">
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>0</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #0D0D0D;
border-top-left-radius: 20px;
border-bottom-left-radius: 20px;</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>30</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QStackedWidget" name="pages">
         <widget class="QWidget" name="mainPage">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <property name="leftMargin">
            <number>20</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QScrollArea" name="scrollArea">
             <property name="styleSheet">
              <string notr="true">QScrollBar:vertical {
    background: transparent;      /* или нужный цвет */
    width: 12px;                  /* толщина скроллбара */
    margin: 0px;                  /* отступы сверху/снизу */
}

/* Ползунок, которым движутся страницы */
QScrollBar::handle:vertical {
    background: #050505;             /* цвет ползунка */
    min-height: 30px;             /* минимальная длина */
    border-radius: 6px;           /* скругляем по полу ширины */
	border: 1.5px solid #1B1B1B;
}

/* Убираем стрелки наверху/внизу */
QScrollBar::sub-line:vertical,
QScrollBar::add-line:vertical {
    height: 0px;
    subcontrol-origin: margin;
    subcontrol-position: top;     /* необязательно */
    background: none;
}

/* Пространство вокруг ползунка (клики) */
QScrollBar::sub-page:vertical,
QScrollBar::add-page:vertical {
    background: none;
}

/* Если нужен горизонтальный тоже — то аналогично */
QScrollBar:horizontal { height:12px; }
QScrollBar::handle:horizontal {
    background:#555; border-radius:6px; min-width:30px;
}</string>
             </property>
             <property name="horizontalScrollBarPolicy">
              <enum>Qt::ScrollBarAlwaysOff</enum>
             </property>
             <property name="widgetResizable">
              <bool>true</bool>
             </property>
             <widget class="QWidget" name="scrollAreaWidgetContents">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>596</width>
                <height>591</height>
               </rect>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_3">
               <property name="spacing">
                <number>0</number>
               </property>
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>40</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <layout class="QGridLayout" name="mainPageGrid">
                 <property name="topMargin">
                  <number>0</number>
                 </property>
                </layout>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="page_2"/>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
